Gruntfile.js

```diff
@@ -1,10 +1,23 @@
 module.exports = function(grunt) {
   'use strict';
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
     eslint: {
       src: ["lib/**/*.js"]
+    },
+    mochaTest: {
+      test: {
+        src: ['lib/**/*-spec.js'],
```

Comment:
[Question] What's the reason why we do not have separate test folder? I do not see any advantages and lots of disadvantages :)